packages/cosmic-swingset/src/block-manager.js

```diff
@@ -44,6 +55,17 @@ export default function makeBlockManager({
       return res;
     };
 
+    if (
+      installationPublisher === undefined &&
+      installationStorageNode !== undefined
+    ) {
+      const marshaller = makeMarshal();
+      const { publisher, subscriber } = makePublishKit();
+      makeStoredSubscriber(subscriber, installationStorageNode, marshaller);
+      installationPublisher = publisher;
+      publisher.publish({ coming: 'soon' });
```

Comment:
```suggestion
      // Publish an inconsequential initial value.
      publisher.publish({ coming: 'soon' });
```

Reply:
This may not be necessary after https://github.com/Agoric/agoric-sdk/pull/6103 and I’ll make a note.